### DISCUSSION TITLE
github action failed with empty log

### DISCUSSION THREAD

**Discussion by liudonghua123** [community/community#25203] [Actions]
<p>I have set a github workflow, but a step in it failed, I checked the log of this step or raw log, it was empty.</p>
<p><a href="https://github.com/liudonghua123/actions/runs/513303118?check_suite_focus=true" rel="nofollow noopener">https://github.com/liudonghua123/actions/runs/513303118?check_suite_focus=true</a></p>
<p>The raw log url is <a href="https://pipelines.actions.githubusercontent.com/gcSqgX7RvlBTO3nj3D3N47j8cRngawJl443qu4217Dy8f0o0vI/_apis/pipelines/1/runs/1/signedlogcontent/3?urlExpires=2020-03-17T09%3A57%3A39.5077358Z&amp;urlSigningMethod=HMACV1&amp;urlSignature=KGYw8ySnEpuKCwUu5LCKjLb6se5FM96K2IqwBhTTkVM%3D" rel="nofollow noopener">https://pipelines.actions.githubusercontent.com/gcSqgX7RvlBTO3nj3D3N47j8cRngawJl443qu4217Dy8f0o0vI/_apis/pipelines/1/runs/1/signedlogcontent/3?urlExpires=2020-03-17T09%3A57%3A39.5077358Z&amp;urlSigningMethod=HMACV1&amp;urlSignature=KGYw8ySnEpuKCwUu5LCKjLb6se5FM96K2IqwBhTTkVM%3D</a></p>
<pre><code>2020-03-17T09:17:38.2295789Z ##[section]Starting: Request a runner to run this job
2020-03-17T09:17:38.3715719Z Requesting a hosted runner in current repository's account/organization with labels: 'ubuntu-latest', require runner match: True
2020-03-17T09:17:38.4515716Z Labels matched hosted runners has been found, waiting for one of them get assigned for this job.
2020-03-17T09:17:38.5578129Z ##[section]Finishing: Request a runner to run this job
</code></pre>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/liudonghua123">@liudonghua123</a> ,</p>
<p>Both links cannot be accessed, is the issue gone? If the workflow logs do not provide enough detail to diagnose why a workflow, job, or step is not working as expected, you can enable additional debug logging: </p>
<p><a href="https://help.github.com/en/actions/configuring-and-managing-workflows/managing-a-workflow-run#enabling-runner-diagnostic-logging" rel="nofollow noopener">Enabling runner diagnostic logging</a></p>
<p><a href="https://help.github.com/en/actions/configuring-and-managing-workflows/managing-a-workflow-run#enabling-step-debug-logging" rel="nofollow noopener">Enabling step debug logging</a></p>
<p>Thanks.</p>
